moveit_core/robot_model/src/link_model.cpp

```diff
@@ -85,25 +85,25 @@ void moveit::core::LinkModel::setGeometry(const std::vector<shapes::ShapeConstPt
          collision_origin_transform_[i].translation().norm() < std::numeric_limits<double>::epsilon()) ?
             1 :
             0;
-    Eigen::Vector3d ei = shapes::computeShapeExtents(shapes_[i].get());
-    Eigen::Vector3d p1 = collision_origin_transform_[i] * (-ei / 2.0);
-    Eigen::Vector3d p2 = collision_origin_transform_[i] * (-p1);
+    Eigen::Vector3d extents = shapes::computeShapeExtents(shapes_[i].get());
```

Comment:
`extends` seems to be used for non-meshes only. Hence computation of shape extends should be moved into the non-meshes `if`-branch below.